addon/components/layers/kml-layer.js

```diff
@@ -0,0 +1,149 @@
+/**
+  @module ember-flexberry-gis
+*/
+
+import Ember from 'ember';
+import BaseLayer from '../base-layer';
+
+/* globals Terraformer, omnivore */
+
+/**
+  Kml layer component for leaflet map.
+
+  @class KmlLayerComponent
+  @extends BaseLayerComponent
+*/
+export default BaseLayer.extend({
+  /**
+    Specific option names available on the Layer settings tab.
+  */
+  leafletOptions: [
+    'kmlUrl',
+    'kmlString'
+  ],
```

Comment:
style is too option of leaflet layer, see wfs layer for example. User should be able to edit layer's appearance like line weight and color.

Reply:
i'll fix it